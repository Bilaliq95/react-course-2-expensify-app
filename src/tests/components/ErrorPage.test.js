import React from 'react'
import ErrorPage from '../../components/ErrorPage';
import {shallow} from 'enzyme'


test('should render error page component correctly',()=>{
    const wrapper=shallow(<ErrorPage />)
    expect(wrapper).toMatchSnapshot();
});