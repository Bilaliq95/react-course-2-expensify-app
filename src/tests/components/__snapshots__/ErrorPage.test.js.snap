// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render error page component correctly 1`] = `
<div>
  404 Not Found
</div>
`;
